### DISCUSSION TITLE
Why I can not load local .glb file?

### DISCUSSION THREAD

**Discussion by DavidShawCn** [google/model-viewer#2306] [Q&A]
`src="https://modelviewer.dev/shared-assets/models/Astronaut.glb"`
its ok to use the sample, but when I use my own glb file, nothing to show.
and my own glb can work on `https://modelviewer.dev/editor/`.
who can help me?

**Comment by ducle8110** [comment]
have same problem here.

**Comment by elalish** [comment]
Most likely the path is wrong; have you checked for console errors? Also note it must be served with proper CORS headers, though that's not usually a problem locally.

**Comment by shamsail** [comment]
Can you give detail which front-end framework are you using? or just testing on the single html file? It would help to find out the structure of your path. It's a path issue locally

**Comment by Turtleted21** [comment]
You need a web server, I use mongoose, it's very easy to use : https://github.com/cesanta/mongoose
you can download pre built binary on the website
